### PR TITLE
feat: get w3c schema details by schema id

### PR DESCRIPTION
**Pull request by bhavanakarwade**
### What?
- Get w3c schemas by schema Id

### Why?
- To fetch W3C type schema details BY schemaId

### CODE REVIEW

**Review by pranalidhanavade — APPROVED** [review]
LG

**Review by pallavighule — APPROVED** [review]
LGTM

**Review by tipusinghaw — APPROVED** [review]

File: apps/api-gateway/src/schema/schema.controller.ts [hunk 1]
```diff
@@ -43,7 +44,7 @@ export class SchemaController {
   async getSchemaById(
     @Res() res: Response,
     @Param('orgId', new ParseUUIDPipe({exceptionFactory: (): Error => { throw new BadRequestException(ResponseMessages.organisation.error.invalidOrgId); }})) orgId: string,    
-    @Param('schemaId') schemaId: string
+    @Param('schemaId', TrimStringParamPipe) schemaId: string
```
Comment (GHkrishna) [inline apps/api-gateway/src/schema/schema.controller.ts:47]: Are we allowing creation of schemas with surrounding spaces?

If yes than will this affect it?

**Review by GHkrishna — CHANGES_REQUESTED** [review]

File: apps/api-gateway/src/schema/schema.controller.ts (hunk 1 above)
Comment (bhavanakarwade) [reply apps/api-gateway/src/schema/schema.controller.ts:47]: Resolved

**Review by GHkrishna — APPROVED** [review]
LG

**Pull request merged**